streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/task/FlinkHttpWatcher.java

```diff
@@ -780,7 +781,7 @@ interface Callback<T, R> {
   /**
    * The situation of abnormal operation alarm is as follows: When the job running mode is yarn per
    * job or yarn application, when the job is abnormal, an alarm will be triggered directly; The job
-   * running mode is yarn session or reome: a. If the flink cluster is not configured with an alarm
+   * running mode is yarn session or remote. If the flink cluster is not configured with an alarm
```

Comment:
From the original text, it should be annotated according to subheadings such as `a`, `b`, and `c`